actions/get.py

```diff
@@ -0,0 +1,11 @@
+from lib.base import BaseRedisAction
+
+__all__ = [
+    'DelRedisAction'
+]
+
+
+class DelRedisAction(BaseRedisAction):
```

Comment:
This class name should change - probably something like `GetRedisAction`

Reply:
Sorry for my mistake.